MekHQ/src/mekhq/campaign/market/contractMarket/AtbMonthlyContractMarket.java

```diff
@@ -498,6 +497,20 @@ public double calculatePaymentMultiplier(Campaign campaign, AtBContract contract
         return multiplier;
     }
 
+    @Override
+    public void checkForFollowup(Campaign campaign, AtBContract contract) {
+        AtBContractType type = contract.getContractType();
+        if (type.isDiversionaryRaid() || type.isReconRaid()
+            || type.isRiotDuty()) {
+            int roll = Compute.d6();
+            if (roll == 6) {
+                addFollowup(campaign, contract);
+                campaign.addReport(
+                    "Your employer has offered a follow-up contract (available on the <a href=\"CONTRACT_MARKET\">contract market</a>).");
```

Comment:
It's really easy for these notices to get missed by the player. Can we either implement this as a dialog, or include selective color coding (and bolded font, for color-blind players)? There are plenty of examples of color-coded dialog in the Education Module, if you need something to base off of.

As a second point, this ideally needs to be added to a ResourceBundle, as per the style guide.

Reply:
So I think those are good suggestions, but this is old code that I simply extracted to the market method and followup contracts aren't implemented in the CO version yet. I'm not opposed to doing it but it feels out of scope for this particular PR since I'm trying to change as little legacy AtB functionality as possible.

Reply:
Ah, didn't spot that! In that case feel free to ignore this part of the review. :)